ansible/group_vars/all/defaults.yaml

```diff
@@ -31,8 +31,7 @@ aws_images: []
 extra_images: []
 
 # FIXME (dkoshkin) use an OS specific and FIPS tar
-containerd_tar_file: cri-containerd-cni-{{ containerd_version }}-linux-amd64.tar.gz
-containerd_url: https://github.com/containerd/containerd/releases/download/v{{ containerd_version }}/{{ containerd_tar_file }}
+containerd_base_url: https://konvoy-image-builder-test.s3.us-west-2.amazonaws.com
```

Comment:
I guess we can get rid of that FIXME 

will we be using a different base url in prod?

Reply:
I'd like to merge into @dkoshkin 's PR first, and start a conversation about this there.